mpc/binance/ecdsa/mpc_test.go

```diff
@@ -108,9 +109,9 @@ func (parties parties) Mapping() map[string]*tss.PartyID {
 }
 
 func TestTSS(t *testing.T) {
```

Comment:
we should add another test which initializes it with secp256k1. 

Perhaps add a loop that iterates over both curve options?

Reply:
I have added a loop to iterate over the curves